docs/source/index.rst

```diff
@@ -38,6 +38,7 @@ cartopy's documentation is arranged userguide form with reference documentation
     crs/index.rst
     crs/projections.rst
     tutorials/using_the_shapereader.rst
+    tutorials/map_styling.rst
```

Comment:
Since this is introducing good practice using the matplotlib interface, I think it makes sense for this to go *after* the matplotlib introduction.